src-tauri/src/node.rs

```diff
@@ -199,7 +199,7 @@ fn create_configuration<CS: ChainSpec + 'static>(
                 other: ExecutionStrategy::NativeElseWasm,
             },
             rpc_http: Some("127.0.0.1:9933".parse().expect("http endpoint is valid")),
```

Comment:
HTTP was introduced in previous PR, but I don't think it needs to be exposed at all

Reply:
true, ok